.gitignore

```diff
@@ -3,3 +3,5 @@
 *.mk
 *.user
 *.d
+src/yquake2.pnproj
+src/yquake2.pnps
```

Comment:
These don't belong in the upstream repo. Editor specific stuff should be in the local ~/.gitignore, otherwise we would need to add support for all editoris in the world...